pkg/kubelet/util/util_windows_test.go

```diff
@@ -22,6 +22,8 @@ package util
 import (
 	"fmt"
 	"math/rand"
+	"net"
+	"os"
```

Comment:
Did this build previously? 😆

Reply:
Apparently not, since we have this issue open: https://github.com/kubernetes/kubernetes/issues/117498

Reply:
Ack, didn't see the referenced issue. Thanks!